backend/bang-ggood/src/main/java/com/bang_ggood/logging/LoggingAspect.java

```diff
@@ -0,0 +1,66 @@
+package com.bang_ggood.logging;
+
+import com.bang_ggood.logging.dto.ErrorLog;
+import com.bang_ggood.logging.dto.InfoLog;
+import com.bang_ggood.logging.dto.WarnLog;
+import jakarta.servlet.http.HttpServletRequest;
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.annotation.AfterReturning;
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.annotation.Before;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import org.springframework.web.context.request.RequestContextHolder;
+import org.springframework.web.context.request.ServletRequestAttributes;
+import java.util.Arrays;
+import java.util.Optional;
+
+@Aspect
+@Component
+public class LoggingAspect {
+
+    private static final Logger log = LoggerFactory.getLogger(LoggingAspect.class);
+
+    @AfterReturning("execution(public * com.bang_ggood..*Service.*(..)))")
+    public void loggingInfo(JoinPoint joinPoint) {
+        if (RequestContextHolder.getRequestAttributes() instanceof ServletRequestAttributes) {
+            HttpServletRequest request = ((ServletRequestAttributes) RequestContextHolder.getRequestAttributes()).getRequest();
+            String methodName = joinPoint.getSignature().getName();
+            InfoLog infoLog = InfoLog.of(request, methodName);
+
+            log.info(infoLog.toString());
```

Comment:
사소한 부분이지만 아래는 요렇게 작성되어 있길래 바꿔봤어요
```suggestion
            log.info(InfoLog.of(request, methodName).toString());
```